backend/src/test/java/com/happy/friendogly/club/service/ClubCommandServiceTest.java

```diff
@@ -92,6 +94,41 @@ void joinClub() {
         assertThat(savedClub.countClubMember()).isEqualTo(2);
     }
 
+    @DisplayName("회원이 모임에 참여 후 정원이 꽉 차면 상태가 Full로 변경된다.")
+    @Transactional
+    @Test
+    void joinClub_StatusFull() {
+        Club club = Club.create(
+                "강아지 산책시키실 분 모아요.",
+                "매주 주말에 정기적으로 산책 모임하실분만",
+                province,
+                city,
+                village,
+                2,
+                savedMember,
+                Set.of(Gender.FEMALE, Gender.FEMALE_NEUTERED),
+                Set.of(SizeType.SMALL),
+                "https://image.com",
+                List.of(savedPet));
+        Club savedClub = clubRepository.save(club);
+
+        Member newMember = Member.builder()
+                .name("위브")
+                .tag("tag123")
+                .build();
+        Member savedNewMember = memberRepository.save(newMember);
+        Pet savedNewMemberPet = createSavedPet(savedNewMember);
+
+        SaveClubMemberRequest request = new SaveClubMemberRequest(List.of(savedNewMemberPet.getId()));
+
+        clubCommandService.joinClub(savedClub.getId(), savedNewMember.getId(), request);
+
+        assertAll(
+                () -> assertThat(savedClub.countClubMember()).isEqualTo(2),
+                () -> assertThat(savedClub.getStatus().isFull()).isTrue()
```

Comment:
```suggestion
                () -> assertThat(savedClub.countClubMember()).isEqualTo(club.getCapacity()),
                () -> assertThat(savedClub.getStatus().isFull()).isTrue()
```

이런식으로 해주면 더 눈에 잘 보일 듯 해요~